### PR TITLE
luci-app-rosy-file-server: add Spanish translation

### PR DESCRIPTION
**Pull request by castillofrancodamian**
Add Spanish translation.

Signed-off-by: Franco Castillo <castillofrancodamian@gmail.com>

### CODE REVIEW

File: applications/luci-app-rosy-file-server/po/es/rosy-file-server.po [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+msgid "<a href='%s%s'>%s</a>"
+msgstr "<a href='%s%s'>%s</a>"
+
+#: applications/luci-app-rosy-file-server/luasrc/model/cbi/rosy-file-server/rosy-file-server.lua:62
+msgid "<a href='%s?server_root=%s%s'>%s</a>"
+msgstr "<a href='%s?Raíz del servidor=%s%s'>%s</a>"
+
+#: applications/luci-app-rosy-file-server/luasrc/model/cbi/rosy-file-server/rosy-file-server.lua:59
+msgid "<a href='%s?server_root=%s'>%s</a>"
+msgstr "<a href='%s?Raíz del servidor=%s'>%s</a>"
```
Comment (feckert) [inline applications/luci-app-rosy-file-server/po/es/rosy-file-server.po]: I think this is wrong.
Translating an URL?

File: applications/luci-app-rosy-file-server/po/es/rosy-file-server.po [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: es\n"
+
+#: applications/luci-app-rosy-file-server/luasrc/model/cbi/rosy-file-server/rosy-file-server.lua:50
+msgid "<a href='%s%s'>%s</a>"
+msgstr "<a href='%s%s'>%s</a>"
+
+#: applications/luci-app-rosy-file-server/luasrc/model/cbi/rosy-file-server/rosy-file-server.lua:62
+msgid "<a href='%s?server_root=%s%s'>%s</a>"
+msgstr "<a href='%s?Raíz del servidor=%s%s'>%s</a>"
```
Comment (feckert) [inline applications/luci-app-rosy-file-server/po/es/rosy-file-server.po]: I think this is wrong.
Translating an URL?

File: applications/luci-app-rosy-file-server/po/es/rosy-file-server.po [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.2.3\n"
+"Last-Translator: Franco Castillo <castillofrancodamian@gmail.com>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: es\n"
+
+#: applications/luci-app-rosy-file-server/luasrc/model/cbi/rosy-file-server/rosy-file-server.lua:50
+msgid "<a href='%s%s'>%s</a>"
+msgstr "<a href='%s%s'>%s</a>"
```
Comment (feckert) [inline applications/luci-app-rosy-file-server/po/es/rosy-file-server.po]: I think this is wrong.
Translating an URL?

File: applications/luci-app-rosy-file-server/po/es/rosy-file-server.po (hunk 1 above)
Comment (castillofrancodamian) [reply applications/luci-app-rosy-file-server/po/es/rosy-file-server.po]: He appeared in poedit to translate

File: applications/luci-app-rosy-file-server/po/es/rosy-file-server.po (hunk 2 above)
Comment (castillofrancodamian) [reply applications/luci-app-rosy-file-server/po/es/rosy-file-server.po]: He appeared in poedit to translate

**Comment by castillofrancodamian** [comment]
He appeared in poedit to translate.

File: applications/luci-app-rosy-file-server/po/es/rosy-file-server.po (hunk 3 above)
Comment (castillofrancodamian) [reply applications/luci-app-rosy-file-server/po/es/rosy-file-server.po]: He appeared in poedit to translate

**Pull request closed without merging**